applications/wg-easy/charts/wg-easy/Chart.yaml

```diff
@@ -1,7 +1,16 @@
-name: wg-easy
-version: 1.0.0
 apiVersion: v2
 dependencies:
-  - name: templates
-    version: '*'
-    repository: file://../templates
+- name: common
+  repository: https://bjw-s-labs.github.io/helm-charts
+  version: 3.7.3
+- name: templates
+  version: '*'
+  repository: file://../templates
+description: Simple wireguard with web configuration management
+home: https://github.com/chris-sanders/helm-charts/charts/wg-easy
+maintainers:
+- email: sanders.chris@gmail.com
+  name: Chris Sanders
+  url: https://github.com/chris-sanders/helm-charts
```

Comment:
This isn't accurate anymore with this PR right?

Reply:
fixt